RoomigrantCompiler/src/main/java/dev/matrix/roomigrant/compiler/data/View.kt

```diff
@@ -0,0 +1,15 @@
+package dev.matrix.roomigrant.compiler.data
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class View(
+		@Json(name = "viewName")
```

Comment:
so every modified file uses spaces and here we have tabs? :)

Reply:
Heh....  Sorry!  I kind of tried to reverse-engineer the existing code style settings to minimize the diff but didn't do a very good job.

Would you be open to setting the code style to per-project and adding it to source control?

Reply:
nvm, I'm using now default formatting rules from AS. thanks tor PR though.